test/unit/storage_sync_test.rb

```diff
@@ -251,22 +237,27 @@ def test_sentinels_empty
 
   private
 
-  def assert_connection(client)
-    client.flushdb
-    client.set('foo', 'bar')
-    assert_equal 'bar', client.get('foo')
-  end
+  def assert_client_config(conn, **conf)
```

Comment:
Following the convention that typically the `assert_` methods follow, I would place the expected config (`**conf` in this case as a second argument, and the actual one (which will get derived from `conn`) as the 2nd argument.

That probably means that a normal hash should be used as the expected param instead of kwargs, but for me this order is a bit confusing... 

Same for `assert_sentinel_config`.

But maybe that's fine...

Reply:
Sound reasonable. I did it in https://github.com/3scale/apisonator/pull/384/commits/65d05ef730d3684e2113b7e206cdad204c639813